### PR TITLE
new cache

### PR DESCRIPTION
**Pull request by biozit**

### CODE REVIEW

**Comment by djw8605** [comment]
Make sure the directory path is NRP too

File: topology/Massachusetts Green High Performance Computing Center/PRP/MGHPCCCachingInfrastructure.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Production: true
+SupportCenter: Community Support Center
+GroupDescription: Caching infrastructure in Nebraska
+GroupID:
+
+Resources:
+ MGHPCC_NRP_OSDF_CACHE:
```
Comment (edquist) [inline topology/Massachusetts Green High Performance Computing Center/PRP/MGHPCCCachingInfrastructure.yaml:7]:
```suggestion
  MGHPCC_NRP_OSDF_CACHE:
```

minor whitespace change

**Comment by biozit** [comment]
on NRP

**Comment by biozit** [comment]
Hi all, do I need to do something else?

**Comment by edquist** [comment]
thanks @biozit - sorry I haven't gotten a chance to take a look at your changes; maybe later today.  I'll let you know!

**Pull request merged**